src/views/setup/Building.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <b-container class="mt-3 align-content-center">
+    <b-row>
+      <h3 class="text-center pb-3">Welcome to the Setup from the COVID-Bouncer Admin</h3>
+    </b-row>
+    <b-row>
+      <form>
+        <b-form-group
+          label="3. Step: Add a Building"
+          id="building"
+          class="input-styling">
+          <b-form-input
+            id="input-building"
+            v-model="building"
+            :state="invalidFeedback"
+            trim>
+          </b-form-input>
+          <b-form-text id="initials-help-block">
```

Comment:
Why cant the "add building" step be skipped? As a User i do not have to add a building.